Streetcode/Streetcode.WebApi/Configuration/RoleAndUserConfiguration.cs

```diff
@@ -0,0 +1,66 @@
+﻿using Microsoft.AspNetCore.Identity;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.AspNetCore.Identity.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using Streetcode.DAL.Persistence;
+using Streetcode.DAL.Entities.Users;
+using Streetcode.DAL.Enums;
+
+namespace Streetcode.WebApi.Configuration
+{
+    public class RoleAndUserConfiguration
+    {
+        public static async Task AddUsersAndRoles(IServiceProvider serviceProvider)
+        {
+            var context = serviceProvider.GetService<StreetcodeDbContext>() !;
+
+            // Create roles in database.
+            await AddRolesAsync(serviceProvider);
+
+            // Populate initial admin with information.
+            var initialAdmin = new User
+            {
+                Name = "admin",
+                Surname = "admin",
+                Email = Environment.GetEnvironmentVariable("ADMIN_EMAIL") !,

```

Comment:
I guess you are overusing !. functionality. It's better for .NET to understand that this in not nullable value instead of telling him directly that you promise him that there won't be a null at runtime by !. syntax

Reply:
I don`t fully understant, what you mean by  following words:
> It's better for .NET to understand that this in not nullable value

P.S. If I remove `!` I get **warning**:
![image](https://github.com/ita-social-projects/StreetCode/assets/113620149/c6fec83a-cbcb-42a0-b5b2-555fa5388873)